t/lib/OpenQA/Test/Utils.pm

```diff
@@ -467,13 +465,14 @@ sub unstable_worker {
             });
         setup_worker($worker, $host);
         $worker->init();
```

Comment:
maybe we can also add a signal handler here and call `Mojo::IOLoop->singleton->stop` on SIGTERM:

```perl

$SIG{TERM} = sub { Mojo::IOLoop->singleton->stop };
```

Then we should reach the `cover()` call reliably later on

Reply:
This seems to break the tests.